.editorconfig

```diff
@@ -0,0 +1,15 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
```

Comment:
Not familiar with editorconfig. Does this line mean applying to all files in this repository?

Reply:
Yes, see if will want to be more conservative and only cover authui v2 code first?